packages/components/src/widgets/BoxedWidget.tsx

```diff
@@ -0,0 +1,19 @@
+import { SquiggleValueChart } from "../components/SquiggleViewer/SquiggleValueChart.js";
+import { valueHasContext } from "../lib/utility.js";
+import { widgetRegistry } from "./registry.js";
+
+widgetRegistry.register("Boxed", {
```

Comment:
I think the whole widget should basically be just the same as the thing it's boxing, but with maybe some small additions (like the title). 

It could be sensible to just have the boxed data passed into all the other Widgets.